### PR TITLE
Update safe-in-cloud

### PR DESCRIPTION
**Pull request by scribblemaniac**

### CODE REVIEW

File: Casks/safe-in-cloud.rb [hunk 1]
```diff
@@ -2,11 +2,15 @@
...
   name 'SafeInCloud'
   name 'SafeInCloud Password Manager'
   homepage 'https://www.safe-in-cloud.com'
-  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
+  license :gratis
 
   app 'Safe In Cloud.app'
+
+  caveats do
+    free_license 'https://www.safe-in-cloud.com/en/license-agreement.html'
```
Comment (vitorgalvao) [inline Casks/safe-in-cloud.rb]: You’re misunderstanding the feature. This is for when a commercial product was discontinued and they provide free licenses for users to use the software without limitations.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 518f3a954235088c2fac927a073b48a92a88f544. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

File: Casks/safe-in-cloud.rb (hunk 1 above)
Comment (scribblemaniac) [inline Casks/safe-in-cloud.rb]: Ah okay, is there a way to include links to custom licenses or should it simply be left out all together?

Comment (vitorgalvao) [inline Casks/safe-in-cloud.rb]: They’re left altogether. `license` is more like a reference field, it doesn’t mean to be completely accurate (which is why `:gpl` is used for variants as well).

Comment (scribblemaniac) [inline Casks/safe-in-cloud.rb]: Okay, thanks!

**Pull request closed without merging**